lib/js/components/Pagination/Pagination.vue

```diff
@@ -0,0 +1,369 @@
+<template>
+	<div
+		:class="{
+			pagination: true,
```

Comment:
I think it might clash with https://versions.bulma.io/0.4.4/documentation/components/pagination/. How about `ds-pagination` similar to https://github.com/bethinkpl/design-system/blob/f8c66e621b659fd96c594ab6adb89c8edbddf061/lib/js/components/Banner/Banner.vue#L2?